File: Selenium/Activities/Activity6.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.common.action_chains import ActionChains
from selenium.webdriver.common.keys import Keys

with webdriver.Firefox() as driver:
    driver.get("https://v1.training-support.net/selenium/input-events")
    print("The Title is :", driver.title)

    action = ActionChains(driver)

    action.send_keys("C").perform()

    action.key_down(Keys.CONTROL).send_keys("a").send_keys("c").key_up(
        Keys.CONTROL
    ).send_keys("a").send_keys("c").pause(3).perform()

    action.send_keys("d").perform()

    action.key_down(Keys.CONTROL).send_keys("a").send_keys("c").key_up(
        Keys.CONTROL
    ).send_keys("a").send_keys("c").pause(3).perform()

    driver.quit()

File: Selenium/Activities/Activity6.java
package activities;

import org.openqa.selenium.Keys;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.interactions.Actions;

public class Activity6 {

	public static void main(String[] args) {

		WebDriver driver = new FirefoxDriver();
		driver.get("https://v1.training-support.net/selenium/input-events");

		System.out.println("The Title of the page is  : " + driver.getTitle());

		Actions act = new Actions(driver);

		act.sendKeys("C").build().perform();

		act.keyDown(Keys.CONTROL).sendKeys("a").sendKeys("c").keyUp(Keys.CONTROL).sendKeys("a").sendKeys("c")
				.pause(1000).build().perform();

		driver.quit();

	}

}
